manifests/repo/debian.pp

```diff
@@ -35,22 +35,22 @@
   Boolean $dotdeb         = true,
   Boolean $sury           = true,
 ) {
-  assert_private()
```

Comment:
This class was supposed to be private. any reason this was changed?

Reply:
Well, I'm not really savvy with puppet... So I converted the module to pdk (since I felt it made it simpler for me by hiding the ruby/gem/rake/whatever that lies behind). And then `pdk test unit --tests=./spec/classes/php_repo_debian_spec.rb` would not work. Maybe it's because my environment is poorly set up? How am I supposed to run the tests without pdk?

Reply:
> How am I supposed to run the tests without pdk?

```
bundle exec rake spec
```

This assume you have a bundle, if not run something like `bundle install --path /tmp/gems` to install gems in a scatch directory without needing to be root

Reply:
Take a look at [CONTRIBUTING](https://github.com/voxpupuli/puppet-php/blob/master/.github/CONTRIBUTING.md) for more info as well